src/pages/storage/actions/CreateStorageBucketBtn.tsx

```diff
@@ -23,7 +23,7 @@ const CreateBucketBtn: FC<Props> = ({ className }) => {
         panelParams.openCreateStorageBucket(project?.name || "");
       }}
       className={className}
-      disabled={!canCreateStorageBuckets(project)}
+      disabled={!canCreateStorageBuckets()}
```

Comment:
```suggestion
      disabled={!canCreateStorageBuckets(project)}
```
This is breaking the permission check.